.github/workflows/commit_skeleton.yml

```diff
@@ -90,10 +90,24 @@ jobs:
       if: ${{ contains(matrix.sgx, 'SGX1') }}
       run: docker exec $rune_test bash -c "rune --debug run skeleton-enclave-container" &
 
+    - name: Wait RA containers Running
+      if: ${{ contains(matrix.sgx, 'SGX1') }}
+      run: |
+      docker exec $rune_test bash -c "count=0;
+      status=$(rune list | grep \"skeleton-enclave-container\" | awk '{print $3}');
+      until [[ \"$status\" = \"running\" ]]
+      do 
+        sleep 5;
+        let count+=1;
+        if [ $count -eq 10 ]; then
+            break;
+        fi
+        status=$(rune list | grep \"skeleton-enclave-container\" | awk '{print $3}');
+      done"
+
```

Comment:
please use  `timeout-minutes: value` to control the maximum time for the check.
line 101 to 104 is no need.
Please refer to 
```yaml
      - name: Check pod runing
        timeout-minutes: 20
        run: |
          sleep $nap_time
          while true; do
            count=$(kubectl get pod | grep "Running" | grep -c "1/1")
            if [ $count -eq 3 ]; then
              break
            fi
            sleep 5
          done
```